### PR TITLE
Bootstrap pagination

### LINKED ISSUE #141: [BUG] Broken pagination on blog
Receive Valuable Information from Our Blog Page 2024-09-21 11-44-54

### PR DESCRIPTION
**Pull request by AucT**
Closes #141

## Summary by CodeRabbit

- **New Features**
	- Introduced a new pagination component with improved styles and functionality.
	- Added pagination-specific styles in a new `pagination.css` file.
	- Enhanced pagination logic with a modular approach using partial templates.
	- Expanded CSS optimization to include new pagination class names.

- **Bug Fixes**
	- Removed outdated pagination styles to prevent conflicts.

- **Documentation**
	- Updated HTML files to link the new pagination stylesheet.

### CODE REVIEW

**Comment by AucT** [comment]
UAObCQ2iJD
LVLgWzDnHs
EsQuwTckRZ

File: postcss.config.js [hunk 1]
```diff
@@ -29,6 +29,9 @@ const purgecss = require("@fullhuman/postcss-purgecss")({
       // pp google form
       // /^pp-gf/, "fl-button", "pp-button",
 
+        ".pagination",
+        ".page-link",
+        ".page-item",
```
[the remark this answers is not in the document]
Comment (pftg) [reply postcss.config.js]: @AucT please follow this advice, it's more correct

[the remark this answers is not in the document]
Comment (AucT) [reply postcss.config.js]: ++

**Comment by pftg** [comment]
@AucT please cleanup PR

**Review by pftg — CHANGES_REQUESTED** [review]
Not working for mobile 
Receive Valuable Information from Our Blog Page 2024-09-26 10-27-07

**Comment by AucT** [comment]
PXiMeCsUdx

File: themes/beaver/layouts/_default/list.html [hunk 2]
```diff
@@ -10,7 +10,8 @@
 | resources.Concat "css/blog.css" | postCSS  | minify | fingerprint | resources.PostProcess}}
 
 <link rel="stylesheet" href="{{ $CSS.RelPermalink }}" media='all' />
-
+{{ $paginationCss := resources.Get "css/pagination.css" | minify | fingerprint }}
```
Comment (pftg) [inline themes/beaver/layouts/_default/list.html:13]: you can add for sure to use: `postCSS | minify | fingerprint | resources.PostProcess`

**Pull request merged**